mne/io/artemis123/artemis123.py

```diff
@@ -32,6 +35,12 @@ def read_raw_artemis123(input_fname, preload=False, verbose=None):
         on the hard drive (slower, requires less memory).
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
+    pos_fname : str or None (defulat None)
+        If not None, load digitized head points from this file
+    head_loc : bool (default False)
+        If True attempt to perform head localization using HPI coils. If no
+        HPI coils are in info['dig'] hpic coils are assumed to be in canonical
+        order of fiducial points (nas, rpa, lpa).
```

Comment:
Yeah I mean the same as CTF in that the coils are at the fiducials and in the same order. If it's not necessarily the same order (I guess it doesn't need to be, really) then no need to mention it.

But it would be good to change the parameter description, and maybe also the name (but I can't think of a good one). `head_loc` is a bit of a misnomer, maybe just `add_trans` if it's indeed computing the `info['dev_head_t']`? Should it default to `True`? If so then it's a form of bugfix so I think it's okay from an API standpoint.

Does it fail gracefully if `True` but no loc is possible?